internal/cmd/platform/delete_platform_test.go

```diff
@@ -40,84 +40,141 @@ var _ = Describe("Delete platforms command test", func() {
-			promptBuffer.WriteString("n")
-			err := executeWithArgs([]string{"platform-name"})
+		When("non-existing platform is being deleted", func() {
+			It("should return message", func() {
+				body := ioutil.NopCloser(bytes.NewReader([]byte("")))
+				expectedError := util.HandleResponseError(&http.Response{Body: body, StatusCode: http.StatusNotFound})
+				client.DeletePlatformsReturns(expectedError)
+				err := executeWithArgs([]string{"non-existing-name", "-f"})
 
-			Expect(err).ShouldNot(HaveOccurred())
-			Expect(buffer.String()).To(ContainSubstring("Delete declined"))
+				Expect(err).ShouldNot(HaveOccurred())
+				Expect(buffer.String()).To(ContainSubstring("Platform(s) not found."))
+			})
 		})
-	})
 
-	Context("when generic parameter flag is used", func() {
-		It("should pass it to SM", func() {
-			client.DeletePlatformsReturns(nil)
-			promptBuffer.WriteString("y")
-			param := "parameterKey=parameterValue"
-			err := executeWithArgs([]string{"platform-name", "--param", param})
-			Expect(err).ShouldNot(HaveOccurred())
+		When("SM returns error", func() {
+			It("should return error message", func() {
+				body := ioutil.NopCloser(bytes.NewReader([]byte("")))
+				expectedError := util.HandleResponseError(&http.Response{Body: body, StatusCode: http.StatusInternalServerError})
+				client.DeletePlatformsReturns(expectedError)
+				err := executeWithArgs([]string{"name", "-f"})
 
-			args := client.DeletePlatformsArgsForCall(0)
+				Expect(err).Should(HaveOccurred())
+				Expect(buffer.String()).To(ContainSubstring("Could not delete platform(s). Reason:"))
 
-			Expect(args.GeneralParams).To(ConsistOf(param))
-			Expect(args.FieldQuery).To(ConsistOf("name eq 'platform-name'"))
-			Expect(args.LabelQuery).To(BeEmpty())
+			})
 		})
-	})
 
-	Context("when non-existing platform is being deleted", func() {
-		It("should return message", func() {
-			body := ioutil.NopCloser(bytes.NewReader([]byte("")))
-			expectedError := util.HandleResponseError(&http.Response{Body: body, StatusCode: http.StatusNotFound})
-			client.DeletePlatformsReturns(expectedError)
-			err := executeWithArgs([]string{"non-existing-name", "-f"})
+		When("no arguments are provided", func() {
+			It("should print required arguments", func() {
+				client.DeletePlatformsReturns(nil)
+				err := executeWithArgs([]string{})
 
-			Expect(err).ShouldNot(HaveOccurred())
-			Expect(buffer.String()).To(ContainSubstring("Platform(s) not found."))
+				Expect(err).Should(HaveOccurred())
+				Expect(err).To(MatchError("single [name] is required"))
+			})
 		})
 	})
 
-	Context("when SM returns error", func() {
-		It("should return error message", func() {
-			body := ioutil.NopCloser(bytes.NewReader([]byte("")))
-			expectedError := util.HandleResponseError(&http.Response{Body: body, StatusCode: http.StatusInternalServerError})
-			client.DeletePlatformsReturns(expectedError)
-			err := executeWithArgs([]string{"name", "-f"})
+	Describe("cascade delete platform", func() {
```

Comment:
add test for trying to delete an active platform

Reply:
Done